setup.sh

```diff
@@ -0,0 +1,31 @@
+#Step 1: sets the output if the script fails for error
+shebang | -ex
+
+#Step2: updates the list of available resources 
+sudo apt-get update -y
+
+#Step3: Install PyEnv Requisites
+sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+#Step3: installs Python Version 
+sudo apt-get install python3-pip
+
+#Step4: Find the Python Version
+python --version
```

Comment:
This isn't really needed to get things working
```suggestion
```

Reply:
Removed the redundant Code